Hindi/hindi.md

```diff
@@ -0,0 +1 @@
+?? ??? ????? ????? ??? ??????? ???? ??? ??? ??? ????? ?? ???? ???? ??? ????? ????? ?? ????? ?????? ???? ?? ??? ?? ??? ??? ????? ??? ?? ???? ??, ?? ?? ????? ????? ??? ??????? ???? ???? ??, ??? ????????? ?? ???? ??? ???? ??? ?? ?? ??? ?? ???? ???
```

Comment:
Oh the problem is with the font I used hindi font in that story

On Tue, Jan 20, 2015 at 2:42 PM, Caroline Buckey <notifications@github.com>
wrote:

> In Hindi/hindi.md
> <https://github.com/udacity/create-your-own-adventure/pull/568#discussion-diff-23249255>
> :
>
> > @@ -0,0 +1 @@
> > +?? ??? ????? ????? ??? ??????? ???? ??? ??? ??? ????? ?? ???? ???? ??? ????? ????? ?? ????? ?????? ???? ?? ??? ?? ??? ??? ????? ??? ?? ???? ??, ?? ?? ????? ????? ??? ??????? ???? ???? ??, ??? ????????? ?? ???? ??? ???? ??? ?? ?? ??? ?? ???? ???
>
> This file doesn't seem to have uploaded correctly. I'm not sure what the
> problem is, but you may want to try to fix that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/udacity/create-your-own-adventure/pull/568/files#r23249255>
> .
>